### ISSUE TITLE
Doesn't show any buses within the next hour

### ISSUE THREAD

**Issue by BeesBoxler** [uni-bus/ios#1]
Ie. The time is 1427 and the app shows the next bus as being at 1540, omitting 1445, 1505 and 1525

**Comment by tombuildsstuff** [comment]
Hey @benjaminboxler - thanks for reporting this.

This looks to be a server/API issue (which is new this year) - I'll try and take a look later today - sorry about this!

**Comment by tombuildsstuff** [comment]
Hey @benjaminboxler

Sorry for the delay getting back to you here (I've been on holiday the last couple of days) - I've found the issue, and unfortunately it's within the iOS app so it needs a new release (where the timezone's being parsed incorrectly).

I've just submitted v2.0.1 to the App Store (and will send an expedite request) - which includes a fix for this - sorry for the inconvenience!

Thanks!
